.github/workflows/overall-tests.yml

```diff
@@ -26,88 +28,142 @@ jobs:
-          title: Taipy Overall Coverage Report
-
-  submit_tests:
-    needs: [partial-tests]
-    timeout-minutes: 20
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11']
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: ${{matrix.python-version}}
-
-      - name: Install pipenv
-        run: curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python
-
-      - name: Install Dependencies
-        run: pipenv install --dev --python=${{ matrix.python-version }}
-
-      - name: Setup LibMagic (MacOS)
-        if: matrix.os == 'macos-latest'
-        run: brew install libmagic
-
-      - name: Pytest Core orchestrator_dispatcher
-        run: pipenv run pytest -m "orchestrator_dispatcher" tests/core
-
-  standalone_tests:
-    needs: [partial-tests]
-    timeout-minutes: 20
-    strategy:
-      fail-fast: false
-      matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11']
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    runs-on: ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@v4
-
-      - uses: actions/setup-python@v5
+      - name: Check if the latest version supported is up to date.
+        id: ensure-dependencies-are-up-to-date
+        working-directory: tools/packages
+        run: |
+          # Ensure dependencies are aligned between Taipy packages
+          pip install -r requirements.txt
+          python check-dependencies.py ensure-same-version
+
+          # Try to update the Pipfile.
+          # Any new packages available are printed to stdout.
+          # If nothing is printed, the Pipfile is up to date and workflow can stop.
+          echo 'diff<<EOF' >> "$GITHUB_OUTPUT"
+          bash check-dependencies.sh ../../Pipfile.sup >> "$GITHUB_OUTPUT"
+          echo EOF >> "$GITHUB_OUTPUT"
+          cat ../../Pipfile.sup
+
+      - name: Create the pull_request
+        if: steps.ensure-dependencies-are-up-to-date.outputs.diff != '' && matrix.python-version == '3.11' && matrix.os == 'ubuntu-latest'
+        uses: peter-evans/create-pull-request@v5
         with:
-          python-version: ${{matrix.python-version}}
-
-      - name: Install pipenv
-        run: curl https://raw.githubusercontent.com/pypa/pipenv/master/get-pipenv.py | python
-
-      - name: Install Dependencies
-        run: pipenv install --dev --python=${{ matrix.python-version }}
-
-      - name: Setup LibMagic (MacOS)
-        if: matrix.os == 'macos-latest'
-        run: brew install libmagic
-
-      - name: Pytest Core standalone
-        run: pipenv run pytest -m "standalone" tests/core
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: Update dependencies
+          branch: dependencies/update
+          base: feature/#414-package-updates
```

Comment:
What??

Reply:
The `base` branch is for tests and will be `develop`. I have no idea for `branch` though.